workflow-engine/src/main/java/com/redhat/parodos/workflows/utils/CredUtils.java

```diff
@@ -0,0 +1,21 @@
+package com.redhat.parodos.workflows.utils;
+
+import java.util.Base64;
+
+public class CredUtils {
```

Comment:
please move this class to parodos-model-api. Don't add any non-workflow-related code in workfow-engine. Btw,  workflow-engine is based on an existing public project: https://github.com/j-easy/easy-flows. Please let's be careful when adding code there. Any base code soley for Parodos should be in parodos-model-api. @pkliczewski @eloycoto @lshannon @anludke @gciavarrini please advise.

Reply:
Thanks for clarifying the differentiation between the engine and model-api.
I moved the class to model-api.